src/scaffold/template/base/src/ts/pet.ts

```diff
@@ -1,37 +1,35 @@
-import {DateOnly} from "apikana/default-types";
```

Comment:
All those changes are necessary because otherwise it would not pass our linting ;-)